sydent/http/servlets/msisdnservlet.py

```diff
@@ -115,8 +115,8 @@ async def render_POST(self, request: Request) -> JsonDict:
                 "errcode": "M_DESTINATION_REJECTED",
                 "error": "Phone numbers in this country are not currently supported",
             }
-        except Exception as e:
-            logger.error("Exception sending SMS: %r", e)
+        except Exception:
+            logger.exception("Exception sending SMS")
```

Comment:
Do we still want the exception type here (or does the whole stack get logged)?

Reply:
We get some of the stack---not 100% sure of the mechanices. We get the locals of each frame that is reportd too (modulo some PII redaction). [Here's](https://sentry.matrix.org/sentry/sydent/issues/22903/?query=is%3Aunresolved) an example of what I see today; I'd like it to look more like [this](https://sentry.matrix.org/sentry/sydent/issues/9423/?query=is%3Aunresolved).

Reply:
Sorry for not being clear -- I meant in the logs, not in Sentry.

Reply:
Ahhh! In that case, you get the whole stack:

```
>>> import logging
>>> logger=logging.getLogger()
>>> try:
...     raise ValueError()
... except Exception:
...     logger.exception("uh oh")
... 
uh oh
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError
```

Reply:
Thanks! Just double checking we still do something reasonable there.

Reply:
Thanks, I appreciate the diligence!